### PR TITLE
[UCAI] Add integration support for LlamaIndex

### PR DESCRIPTION
**Pull request by BenWilson2**
**PR Checklist**

- [X] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

Adds support for LlamaIndex to the unity catalog AI integrations.

### CODE REVIEW

File: unitycatalog-ai/integrations/llama_index/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+pip install ucai-llamaindex
+```
+
+### From source
+
+To get started with the latest version, you can directly install this package from source via:
+
+<!-- TODO: update this to the actual path where the repo's main branch will live -->
+```sh
+pip install git+https://github.com/BenWilson2/unitycatalog-ai.git@llama-tool#subdirectory=integrations/llama_index
```
Comment (serena-ruan) [inline unitycatalog-ai/integrations/llama_index/README.md:19]:
```suggestion
pip install git+https://github.com/unitycatalog/unitycatalog.git#subdirectory=unitycatalog-ai/integrations/llama_index
```

File: unitycatalog-ai/integrations/llama_index/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+```sh
+pip install ucai-llamaindex
+```
+
+### From source
+
+To get started with the latest version, you can directly install this package from source via:
+
+<!-- TODO: update this to the actual path where the repo's main branch will live -->
```
Comment (serena-ruan) [inline unitycatalog-ai/integrations/llama_index/README.md:17]:
```suggestion
```

**Review by serena-ruan — APPROVED** [review]
LGTM! Just small updates in RADEME :)

**Comment by BenWilson2** [comment]
@dennyglee this one's all set to merge!

**Review by dennyglee — APPROVED** [review]

**Pull request merged**